linera-base/src/identifiers.rs

```diff
@@ -203,14 +193,6 @@ pub struct BlobId {
 }
 
 impl BlobId {
```

Comment:
If we want to support queries like "list all the bytecodes" in DB, then the blob type should be first.

This may change the profile of ScyllaDb sharding but I trust that we can optimize this separately. @MathieuDutSik What do you think?

Reply:
Done in https://github.com/linera-io/linera-protocol/pull/3062/commits/6bb25ed17eeabb432ff022d57f1bc3ebd8d34539.